otp.py

```diff
@@ -132,12 +133,15 @@ def main():
     parser = argparse.ArgumentParser(description='Compute OTP strings.')
     parser.add_argument('--test', help='Run the test suite.', nargs=0,
                         action=RunTests)
+    parser.add_argument('args', nargs=argparse.REMAINDER)
   
     # Note: this may exit, if tests are run.
-    _ = parser.parse_args()
+    parsed = parser.parse_args()
 
-    line = input('Challenge? ')
-    parts = line.split()
+    parts = parsed.args
+    if len(parts) != 3:
```

Comment:
some of our servers issue challenges with a fourth arg "ext". I think the program should allow a cut/paste that includes that arg, rather than demanding the user to carefully cut/paste just 3 args.